gtfu/src/main/java/gtfu/Recipients.java

```diff
@@ -0,0 +1,33 @@
+package gtfu;
+import java.io.*;
+import java.util.*;
+
+public class Recipients {
+
+	private static final String RECIPIENTS_FILE_PATH = "src/main/resources/conf/recipients.txt";
+	private static final String DELIMITER = ":";
+	private Map<String, String> recipientsMap;
+
+	public Recipients() {
+
+        recipientsMap = Util.hashMapFromTextFile(RECIPIENTS_FILE_PATH,DELIMITER);
+	}
+
+    public String[] get(String reportType) {
+
+        String recipientsList = null;
+        // Default to local recipient list if env variable is present
+        recipientsList = System.getenv("LOCAL_REPORT_RECIPIENTS");
```

Comment:
There isn't really a point to initializing to `null` if you're reassigning in the very next step. `System.getenv()` returns `null` for unset environment variables.